hardware/raspberrypi/README.md

```diff
@@ -12,7 +12,7 @@ The hardware in the Raspberry Pi
     - The Broadcom processor used in Raspberry Pi 3 (and later Raspberry Pi 2)
 - [BCM2837B0](bcm2837b0/README.md)
     - The Broadcom processor used in Raspberry Pi 3B+ and 3A+
-- [Bootmodes](bootmodes/README.md)
+- [Boot modes](bootmodes/README.md)
     - A description of the available BCM2835/6/7 bootmodes
```

Comment:
Should this PR put a space in this line too? :wink:

Reply:
Ooh good catch. We're fairly battering through them tonight 😁